storages/python/tests-python/test_http.py

```diff
@@ -0,0 +1,13 @@
+try:
+    import requests
+except ModuleNotFoundError:
```

Comment:
I don't like this since it makes it very easy to miss out if things are working or not in ci/cd. Our ci/cd workflows should install all requirements and test everything.

It is hard to see the output of the workflow here, it just passes, but it seems to me that requests is still not added to the requirements, which means that the test is passing without having even tried and it should fail.

Reply:
I double-checked this. If I uninstall requests, ctest will report test_http as skipped, not as passed.

I also added requests to requirements.txt, so that we ensure that test_http is run by CI/CD on github.